components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/util/RemoteFetchConfigurationUtils.java

```diff
@@ -71,4 +77,48 @@ public static String generateUniqueID() {
 
         return UUID.randomUUID().toString();
     }
+
+    /**
+     * Get the Default Items per Page needed to display.
+     *
+     * @return defaultItemsPerPage need to display.
+     */
+    public static int getDefaultItemsPerPage() {
```

Comment:
We are reading and parsing this per each request. Need to do it only once at the initialisation time